recipes/ssm-simulators/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "ssm-simulators" %}
+{% set version = "0.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ssm-simulators-{{ version }}.tar.gz
+  sha256: 0d08b4a8747fc39d48db863211eccb3608d2731e961de5afbcaee16c7f242ef2
+
+build:
+  skip: true  # [py>=312 or py2k]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+  host:
+    - python
+    - setuptools
+    - wheel
+    - cython >=0.29.23
+    - numpy >=1.20
```

Please omit the pin so that the global numpy pin is taken.
```suggestion
    - numpy
```

---

Done